press/press/doctype/deploy_candidate/deploy_candidate.py

```diff
@@ -250,6 +250,12 @@ def _prepare_build_context(self):
 		dockerfile = os.path.join(self.build_directory, "Dockerfile")
 		with open(dockerfile, "w") as f:
 			dockerfile_template = "press/docker/Dockerfile"
+
+			for d in self.dependencies:
+				if d.dependency == "BENCH_VERSION":
+					if d.version != "5.2.1":
```

Comment:
This can be merged in single line `if d.dependency == "BENCH_VERSION" and d.version != "5.2.1":`